blog/2024-08-19-join-us-for-wasmcloud-innovation-day/index.mdx

```diff
@@ -0,0 +1,75 @@
+---
+title: "Join us for wasmCloud Innovation Day!"
```

Comment:
The title feels a bit weird when in preview. Suggest (think I prefer 2 or 3)

wasmCloud Innovation Day is coming!
Join us! wasmCloud Innovation Day
Announcing wasmCloud Innovation Day!

Reply:
These all had orphaned lines either on the blog page or post page or both as well -- suggesting an alternative in my commit but open to others as well